src/DataStructures/Tensor/EagerMath/ScaleBy.hpp

```diff
@@ -0,0 +1,28 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function scale_by
+
+#pragma once
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+
+/*!
+ * \ingroup Functions
+ * \brief Divides the components of a tensor by a scalar
+ *
+ * \returns a tensor of the same type as the input tensor
+ */
+template <typename TensorType>
+TensorType scale_by(TensorType tensor, const DataVector& scale) {
```

Comment:
Let's just rename it since to me `scale_by` means multiply, since you'll have to make other changes anyway.